### DISCUSSION TITLE
Expedition 12 Phase 3 Mission 4 Bug

### DISCUSSION THREAD

**Discussion by LoverBoyUSA** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#12] [General]
Hot Pursuit mission will not complete. I have tried it 6 times now.

**Comment by cwmonkey** [comment]
Looking into this now.

**Comment by cwmonkey** [comment]
@LoverBoyUSA This seemed to work fine for me. My exact steps were:

1. I built a base on some random planet in a non-pirate system
2. I warped to a pirate planet with my hyperdrive
3. I landed on the pirate space station and bought a bunch of contraband
4. I portaled back to my base and with the mission selected I sold some goods at a trade terminal (just as a test)
5. I flew to the space station and sold the rest of my contraband and the quest completed as normal

Was any of that different from what you were trying?

![275850_20240802110119_1](https://github.com/user-attachments/assets/53f5f402-f013-4026-8ea1-1e15b2be24af)
![275850_20240802110246_1](https://github.com/user-attachments/assets/dbbc9364-91ef-473b-860f-fa2d2a31edba)
![275850_20240802110300_1](https://github.com/user-attachments/assets/67a3281c-b839-469c-8993-26d15fbaadf7)
![275850_20240802110305_1](https://github.com/user-attachments/assets/8ac51ab5-3c24-409f-855a-5a9596bc425a)
![275850_20240802110326_1](https://github.com/user-attachments/assets/a203225d-7162-4ac0-8c38-78f5a4042b76)

**Reply by cwmonkey** [reply]
Oh and I used the json file from my site: https://cwmonkey.github.io/nms-expeditions/

But that shouldn't be any different than the one here.
